### DISCUSSION TITLE
Link to services in the alert history entries

### DISCUSSION THREAD

**Discussion by CharlieHird** [chriscareycode/nagiostv-react#52] [Ideas]
I was wondering if it would be possible to add a link to the alert history entries, below the graphs, to their respective services in the same manner the current warnings/alerts do? If you do not think there would be any interest for other users I can just change mine!
Thanks,
Charlie

(Chris, turns out the issue was a browser based error. Moved discussion here...)

**Comment by chriscareycode** [comment]
Fix for this is done and will go out with the next release

**Reply by CharlieHird** [reply]
Excellent!
Thank you

**Comment by chriscareycode** [comment]
You can try out the new v0.8.0 release and let me know how that's working for you https://github.com/chriscareycode/nagiostv-react/releases

**Reply by CharlieHird** [reply]
Updated and it is working great.
Thanks for the effort!
